languages/julia/exercises/concept/fibonacci-iterator/.docs/instructions.md

```diff
@@ -11,11 +11,8 @@ The start of the sequence is $1,~1,~2,~3,~5,~8,~13,~21,~34,~55,~89,~\dots$
 In maths notation: <!-- TODO: Move this to an info box once they're available -->
 
 $$
-
```

Comment:
@SaschaMann ?

Reply:
Yea whatever, let prettier do its thing. I thought this had been fixed by #2845 already though